go.mod

```diff
@@ -2,6 +2,16 @@ module github.com/gravitational/teleport
 
 go 1.25.1
 
+ignore (
+	build
+	docs
+	node_modules
+	rfd
+	target
+	web
```

Comment:
Should this be `./web`?

From https://go.dev/ref/mod#go-mod-file-ignore

> If the path starts with ./, the path is interpreted relative to the module root directory, and that directory and any directories or files recursively contained in it will be ignored when matching package patterns.